pallets/ctype/src/benchmarking.rs

```diff
@@ -57,10 +58,32 @@ benchmarks! {
 
 	}: _<T::Origin>(origin, ctype)
 	verify {
-		let stored_ctype_creator: T::CtypeCreatorId = Ctypes::<T>::get(ctype_hash).expect("CType hash should be present on chain.");
+		let stored_ctype_entry = Ctypes::<T>::get(ctype_hash).expect("CType hash should be present on chain.");
 
 		// Verify the CType has the right owner
-		assert_eq!(stored_ctype_creator, did);
+		assert_eq!(stored_ctype_entry.creator, did);
+	}
+
+	set_block_number {
+		let caller = account("caller", 0, SEED);
+		let did: T::CtypeCreatorId = account("did", 0, SEED);
+
+		let ctype: Vec<u8> = (0u8..u8::MAX).cycle().take(MAX_CTYPE_SIZE.try_into().unwrap()).collect();
```

Comment:
I wonder if worst case behaviour would rather be having all bytes at max value?

Reply:
You mean all bytes with value 255? What would that change?

Reply:
Not sure, it could have a tiny tiny tiny impact on decoding/encoding times?  😅 Regard this as irrelevant.

Reply:
I am fairly certain that decoding a boolean takes the same time a decoding a byte, and that the value of the bits do not matter 😊